common-pipeline/operators/uihints-v3-schema.json

```diff
@@ -456,6 +457,14 @@
         "class_name": {
           "description": "Optional class name to set for this control",
           "type": "string"
+        },
+        "dropdown_selection": {
```

Comment:
Why do we need this?  Shouldn't the new control type determine this?

Reply:
I was thinking that this new control will be similar to how `oneofselect` is determined. Since this is still a dropdown, it can still be identified via an enum list of 5 or more items with an extra property to indicate multiple selection for this dropdown. At the same time, specifying the control type will also determine this multiselect dropdown.
So basically this isn't really needed. Should I remove it?

Reply:
Right, I think you can remove it.